### PR TITLE
WhatsApp(Independent Publisher)

### PR DESCRIPTION
**Pull request by zak0807**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [x] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [x] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [x] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [x] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [x] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md. 

Flow
Flow_test_success

Test_success
Paconn_validate_success

### CODE REVIEW

File: independent-publisher-connectors/WhatsApp/ReadME.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+By default Meta uses a temporary token that expires in 24 hours. This temporary token can be used as the API key but will have to be updated within 24 hours.
+
+To get a permanent token, follow the steps from the following link: [Permanent token for WhatsApp Cloud API: How to generate? | Pepper Cloud Blog](https://blog.peppercloud.com/generate-permanent-token-for-whatsapp-cloud-api/)
+
+## Known Issues And Limitations
+
+Recipient is unable to reply to the messages as a webhook hasn’t been configured on the connector.
+
+For now the connector can only be used in a Power Automate Flow as Power Apps doesn’t recognise the innermost nested arrays found in the JSON body. By setting the raw body, it would work. In Power Automate, it works perfectly.
```
Comment (Amjed-Ayoub) [inline independent-publisher-connectors/WhatsApp/ReadME.md:58]: @zak0807 all file should be free of grammatical errors and typos. here you have "...recognise...", can you please change to "...recognize...".

**Review by Amjed-Ayoub — CHANGES_REQUESTED** [review]
@zak0807 Thank you so much for your submission. May I ask you please to remove all the images from the PR files. You already have them above in the PR description and that is enough :-).
image

Please let me know if you have any questions.

**Comment by zak0807** [comment]
> @zak0807 Thank you so much for your submission. May I ask you please to remove all the images from the PR files. You already have them above in the PR description and that is enough :-). image
> 
> Please let me know if you have any questions.

Hi Amjed. I've been trying to remove the files that you told me to but somehow I'm getting this message when doing so: "This commit does not belong to any branch on this repository, and may belong to a fork outside of the repository."

What should I do in order to remove the images?

Thanks!

**Comment by Amjed-Ayoub** [comment]
> > @zak0807 Thank you so much for your submission. May I ask you please to remove all the images from the PR files. You already have them above in the PR description and that is enough :-). image
> > Please let me know if you have any questions.
> 
> Hi Amjed. I've been trying to remove the files that you told me to but somehow I'm getting this message when doing so: "This commit does not belong to any branch on this repository, and may belong to a fork outside of the repository."
> 
> What should I do in order to remove the images?
> 
> Thanks!

@zak0807 It should be the same way you removed the settings.json file. but you can close this PR and create a new one with the required files only. please see below.

1. apiDefinition.swagger.json
2. apiProperties.json
3. readme.md

Thank you so much.

**Pull request closed without merging**